aws/cloudformation/metaflow-cfn-template.yml

```diff
@@ -88,18 +112,27 @@ Mappings:
     Role:
       value: ""
 Conditions:
+  CreateOwnVpc: !Or
```

Comment:
After some testing, this seems a little bit opinionated.  In short, the conditional effectively says "_If any of these are blank, ignore them all._"  While that makes sense for VPC ID, CIDR, and security group, the number and quantity of subnets is likely to vary.  Just to pick out an example, this conditional doesn't seem to apply to VPC's with no private subnets or more than 2 public subnets.